democracy/tests/test_images.py

```diff
@@ -1,12 +1,12 @@
 import pytest
 
-from democracy.tests.test_hearing import list_endpoint
-from democracy.tests.utils import IMAGES, get_data_from_response, get_hearing_detail_url
+from democracy.tests.utils import IMAGES, create_default_images, get_data_from_response, get_hearing_detail_url
 
 
-def check_entity_images(entity):
-    assert 'images' in entity
-    image_list = entity['images']
+def check_entity_images(entity, images_field=True):
```

Comment:
Would just be better to directly pass `entity['images']` instead of having this peel it off?